NATIVE_POSTGRES_JSON_COMPARED_TO_EQL.md

```diff
@@ -0,0 +1,510 @@
+# Native Postgres JSON(B) Compared to EQL
+
+EQL supports a subset of functionality supported by the native Postgres JSON(B) functions and operators. The following examples compare native Postres JSON(B) functions and operators to the related functionality in EQL.
+
+## `json ->> text` → `text` and `json -> text` → `jsonb`/`json`
+
+### Native Postgres JSON(B)
+
+```sql
+-- `->` (returns JSON(B))
+SELECT plaintext_jsonb->'field_a' FROM examples;
+
+-- `->>` (returns text)
+SELECT plaintext_jsonb->>'field_a' FROM examples;
+```
+
+### EQL
+
+EQL JSONB functions accept an eJSONPath as an argument (instead of using `->`/`->>`) for lookups.
```

Comment:
We should either define eJSONPath in this document, or link to it elsewhere in this repo.

Reply:
Fixed in https://github.com/cipherstash/encrypt-query-language/pull/61/commits/beabf2bd22ddd7489755bcb60e25aaaadc48bc20 and https://github.com/cipherstash/encrypt-query-language/pull/61/commits/9224ad68fcc389a792f253397b66f6287f2205c8.